cmd/grpc-client-cli/main.go

```diff
@@ -129,6 +129,12 @@ func main() {
 			Usage:       `If set, send keepalive pings every "keepalive-time" timeout. If not set, default grpc settings are used`,
 			DefaultText: "not set",
 		},
+		&cli.IntFlag{
+			Name: "max-receive-message-size",
```

Comment:
@jhaygood86 I guess `max-send-message-size` is not needed yet, but we should probably add it at some point too